Utilities/KWStyle/KWStyle.cmake

```diff
@@ -4,10 +4,10 @@ if(ITK_USE_KWSTYLE)
     ${ITK_SOURCE_DIR}/Utilities/KWStyle/ITK.kws.xml
     )
   set(kwstyle_itk_examples_files_list_file
-    ${ITK_BINARY_DIR}/Utilities/KWStyle/ITKExamplesFiles.txt
+    ${ITK_BINARY_DIR}/Utilities/KWStyle/ITKSphinxExamplesFiles.txt
     )
   configure_file( # KWStyle requires that the files list be absolute paths
-    ${ITK_SOURCE_DIR}/Utilities/KWStyle/ITKExamplesFiles.txt.in
+    ${ITK_SOURCE_DIR}/Utilities/KWStyle/ITKSphinxExamplesFiles.txt.in
```

Comment:
Same comment here: see [ITKExamplesFiles.txt.in](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKExamplesFiles.txt.in).

Reply:
Done